client/my-sites/site-settings/wrap-settings-form.jsx

```diff
@@ -30,7 +30,7 @@ import {
 } from 'state/selectors';
```

Comment:
Seems like we also need to update the `isUpdatingJetpackSettings` and `isJetpackSettingsSaveFailure` selectors the same way we did with `isRequestingJetpackSettings`, otherwise we might be introducing subtle regressions.

Reply:
Good catch. I'll grep for `jetpack.settings` to see what other selectors might be using that part of the state tree.

Reply:
The `isUpdatingJetpackSettings` selector could be tricky, since it requires a fully instantiated action object -- i.e. we need to pass a specific `settings` object to it, and it will only return `true` if _those_ are the settings that are currently being updated. That might not be totally trivial to feed into `WrapSettingsForm`'s `connect`...

Reply:
I I hope that the following will work:

```diff
diff --git a/client/my-sites/site-settings/wrap-settings-form.jsx b/client/my-sites/site-settings/wrap-settings-form.jsx
index 9ce194a..64812b1 100644
--- a/client/my-sites/site-settings/wrap-settings-form.jsx
+++ b/client/my-sites/site-settings/wrap-settings-form.jsx
@@ -329,7 +330,7 @@ const wrapSettingsForm = getFormSettings => SettingsForm => {
                }
        );
 
-       return flowRight( connectComponent, localize, trackForm, protectForm )( WrappedSettingsForm );
+       return flowRight( trackForm, protectForm, connectComponent, localize )( WrappedSettingsForm );
 };
 
 export default wrapSettingsForm;
```

(That way, the `fields` prop will be available to our `connectComponent()` function, where we can then hopefully build a proper `isUpdatingJetpackSettings` check.)

I'll probably do that in a different PR so we can test it thoroughly.

Reply:
Sounds sensible 👍